docs/spark/tutorials/get-started.md

```diff
@@ -209,14 +229,14 @@ This .NET app counts words with Apache Spark
    dotnet build
    ```
 
-2. Run the following command to submit your application to run on Apache Spark:
+2. Navigate to your build output directory and enter the following command to submit your application to run on Apache Spark:
```

Comment:
The example below is using bash style line continuations. This code doesn't copy\paste to windows unless someone is using a bash terminal. This doesn't work on powershell or cmd.exe